tendrl/commons/objects/node/atoms/is_node_tendrl_managed/__init__.py

```diff
@@ -52,7 +52,7 @@ def run(self):
 
             # Check if node has the block devices populated
             try:
-                block_devices = NS._int.client.read("nodes/%s/BlockDevices" % node_id)
+                block_devices = NS._int.client.read("nodes/%s/LocalStorage/BlockDevices" % node_id)
```

Comment:
and you missed LocalStorage/virtio 

We should not be doing these disk checks at all, because we cannot be sure if its going to show up as virtio or blockdevice or disk

I am removing the disk checks for now